core/common/src/main/java/alluxio/exception/AlluxioException.java

```diff
@@ -20,7 +20,7 @@
  * the RPC framework and convert back without losing any necessary information.
  */
 @ThreadSafe
-public abstract class AlluxioException extends Exception {
+public class AlluxioException extends Exception {
```

Comment:
I actually prefer `AlluxioRuntimeException`, because it carries more information, i.e. it's converted from RTE